website/content/vagrant-cloud/hcp-vagrant/migration-guide.mdx

```diff
@@ -0,0 +1,149 @@
+
+To auto-populate the migration tool with a specific Vagrant Cloud `Organization` click on the `Migrate to HCP` button in the top right corner of your [settings](https://app.vagrantup.com/settings) page .
+
+#### 2. Select a Vagrant Cloud organization to migrate
+
+Select a single organization to migrate or select the `Migrate all organizations` checkbox to migrate them all at once to a single HCP Project.
+
+You may select any of the organizations listed in the Vagrant Cloud organization drop-down, even if a specific organization was pre-populated after navigating to the migration tool from your [Settings](https://app.vagrantup.com/settings) page.
+
+##### 3. Select a destination project in HCP
+
+If you have multiple projects in HCP, select the desired project from the HCP Project drop-down. If you have not created a second project the `default-project` will be auto-selected.
+
+When preparing for migration it is worth noting the following:
+
+- Your Vagrant Cloud Organization will be put into a read-only state and marked `Migrating` until the migration is complete. During this time boxes will be available in searches and to download, but write functionality like uploading new boxes, releasing versions, and managing settings for your Vagrant Cloud Organization will be unavailable.
+- Your new HCP Registry will not be available until the migration is complete.
+- When you migrate your default organization your Profile Settings (user name, gravatar url, etc) will no longer be available to manage in Vagrant Cloud and should be managed from your [HCP Settings](https://portal.cloud.hashicorp.com/account-settings).
+
+#### 4. Start the migration
+
+After you press the `Migrate` button you will be redirected to a status page that will keep you updated on the progress of your migration.
+
+#### 5. Visit your Registry in HCP
+
+Once your migration is complete you can view your new Registry in HCP!
+
+You can see your registries by clicking on[ Vagrant Registry](https://portal.cloud.hashicorp.com/services/vagrant/registries) in the side panel or by visiting [https://portal.cloud.hashicorp.com/services/vagrant/registries](https://portal.cloud.hashicorp.com/services/vagrant/registries).
+
+It is important to note that while a migrated organization will no longer be available in Vagrant Cloud the status of the migration will be visible from the `Status` tab on the [Migration](https://app.vagrantup.com/migration) page.
+
+## Conclusion
+
+If for some reason you are unable to access your new HCP Registry or your migration does not complete in a timely fashion check our [Troubleshooting Guide ](/vagrant/vagrant-cloud/hcp-vagrant/troubleshooting)or contact us at [support@hashicorp.com](mailto:support@hashicorp.com) with the subject `Vagrant Migration`. Our team will be happy to help you complete your migration or reset your organization state.
+
```

Comment:
Can you delete this white space at the end?

Reply:
I installed the weirdest super-linter and it has been driving me crazy. sorry about this.